recipes/sfml/all/conanfile.py

```diff
@@ -91,10 +107,12 @@ def generate(self):
         tc.variables["SFML_BUILD_AUDIO"] = self.options.audio
         tc.variables["SFML_INSTALL_PKGCONFIG_FILES"] = False
         tc.variables["SFML_GENERATE_PDB"] = False
-        tc.variables["SFML_USE_SYSTEM_DEPS"] = True
+        tc.variables["SFML_USE_SYSTEM_DEPS"] = Version(self.version) < "3.0.0"
         tc.variables["WARNINGS_AS_ERRORS"] = False
-        if Version(self.version) >= "2.6.0":
-            tc.variables["CMAKE_CXX_STANDARD"] = 11
+
+        if self.settings.compiler.cppstd is not None:
+            tc.cache_variables["CMAKE_CXX_STANDARD"] = self._min_cppstd
```

Comment:
There is no need to set this. The CMake script already uses `target_compile_features` to set the proper C++ standard version.

Reply:
It does not build for MSVC without (which is what lead to adding this)